src/customers/account-create.md

```diff
@@ -150,7 +150,20 @@ Customer accounts created from the Admin do not initially have passwords assigne
 
 1. In the menu bar at the top of the page, click **Reset Password**.
 
-1. Notification is sent to the account owner, with instructions for setting the password.
+   The number of password reset requests that are allowed within an hour is set in the [configuration]({% link configuration/customers/customer-configuration.md %}).
+
+## Revoke a customer’s OAuth tokens
+
+{:.bs-callout-warning}
+Do not proceed unless you are a developer familiar with API Authentication.
```

Comment:
Why is this warning necessary?

Reply:
Hi @dobooth 😊 thank you for reviewing!

I have noticed, the same warning on the resetting password topic, and added to this one as well. If you think that this is mistaken, let's remove this for both topics, so as not to confuse users studying documentation from different angles
![image](https://user-images.githubusercontent.com/78729171/116310886-3e3b7280-a7b3-11eb-85a1-042dcdc769b3.png)

Reply:
Let's remove the warning. Most users here will not be devs with API Auth experience but we don't say why that is needed. In most cases, the user will just then login again as usual. No need to touch the password file.